contracts/arbitrum/connectors/dsa-spell/events.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma abicoder v2;
+
+contract Events {
+	event LogCastOnDSA(
+		address indexed targetDSA,
+		string[] connectors,
+		bytes[] datas
+	);
+	event LogCastAny(
+		string indexed connector,
```

Comment:
@Richa-iitr why connector and connectorName?

Reply:
Added just in case we need to ever track/filter events by connector name. Should i remove it if not needed?

Reply:
@Richa-iitr I think connector and connector name will be same, why adding it two times?

Reply:
I added one for the topics(hashed form) and one for the data(string form) in the transaction events. Removed the indexed one.